### PR TITLE
added profile information to profile detail page

### PR DESCRIPTION
**Pull request by oware**
profile information displays if user is logged in. commit resolves issue #1895

### CODE REVIEW

**Comment by oware** [comment]
allowed document types listed. commit resolves #1792

File: geonode/documents/templates/documents/document_upload_base.html [hunk 1]
```diff
@@ -10,6 +10,7 @@
 <div class="page-header">
   <a href="{% url "document_browse" %}" class="btn btn-primary pull-right">{% trans "Explore Documents" %}</a>
   <h2 class="page-title">{% trans "Upload Documents" %}</h2>
+  <p>Allowed document types [doc, docx, gif, jpg, jpeg, ods, odt, odp, pdf, png, ppt, pptx, rar, tif, tiff, txt, xls, xlsx, xml, zip, gz]
```
Comment (ischneider) [inline geonode/documents/templates/documents/document_upload_base.html]: Any way to get this list from the actual code/configuration?

Comment (capooti) [inline geonode/documents/templates/documents/document_upload_base.html]: I was thinking the same, but I don't think there is a better way other than setting this in a context variable or from context processor.
Does such an overhead justifies the effort?

**Comment by garnertb** [comment]
I think so, since the admin can arbitrarily restrict this at anytime and the template will be incorrect.

File: geonode/documents/templates/documents/document_upload_base.html (hunk 1 above)
Comment (ischneider) [inline geonode/documents/templates/documents/document_upload_base.html]: > I was thinking the same, but I don't think there is a better way other than setting this in a context variable or from context processor.

Using a templatetag would be the most straight-forward.

A simple tag could just spit out the list of extensions (as configured in settings/wherever).

**Comment by jj0hns0n** [comment]
This PR has a bunch of extraneous stuff. @oware can you close and resubmit in a more atomic way.

**Pull request closed without merging**